a.sh

```
@@ -0,0 +1,87 @@
+
+export OP_PROVER=0x4Dba9439caE5B8fC4EC881d73738c13545E6ecc8
+export BASE_PROVER=0x4Dba9439caE5B8fC4EC881d73738c13545E6ecc8
+export SETTLEMENT_REGISTRY=0xa102b8A5F1a46C0C3D516a9b2237F0D886B21479
+export OP_NATIVE_PROVER=0xAf35b23F38fe8eE793AA6E03d7D54c93c8a6F4B3
+export BASE_NATIVE_PROVER=0x63C0d99458E2d2153cF95ad8b800e191633F0d69
+
+
+cd $CLI_PATH && make build 
+OUTFILE=~/Code/prover-contracts/test/native_fallback/payload/native-proof.hex
+$CLI_PATH/bin/native-proof proveNative \
+    --src-l2-chain-id $(cast chain-id -r $OP_RPC_URL) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)